.github/workflows/docs.yml

```diff
@@ -98,3 +98,21 @@ jobs:
           path: /tmp/pr_number.txt
           retention-days: 1
           if-no-files-found: error
+
+  r:
+    if: github.event_name != 'pull_request' || github.event.pull_request.draft == false
+    permissions:
+      contents: read
+    timeout-minutes: 15
```

Comment:
15 minutes should be fine, I checked previous CircleCI runs and it's around 3m on average.